### PR TITLE
[10.0] Migrate website cookie notice

### PR DESCRIPTION
**Pull request by hurrinico**
WIP until:

- [x] #353

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Wow, what a huge commit quantity. Can you please squash by author and all the translation one together?

**Comment by hurrinico** [comment]
depends on https://github.com/OCA/website/pull/310 , sorry for closing and reopen, i've made a mistake

**Comment by hurrinico** [comment]
Hi @pedrobaeza i've made this pr following strictly [this guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0) and i think that all that commit comes from 9.0 version, so why squash them?.

**Comment by pedrobaeza** [comment]
I know that you have followed the guide (and I have to reflect this somehow in the guide), but we don't need to keep such commit detail in this branch as the important part are the authors and the 10.0 part. For example, if during the 8.0 life cycle, you have retranslated 20 times the module terms, why keep all these diffs to increase a lot the branch size (and thus the download/pull size). We only need the last translation. The same can apply to the rest of the code, but with one exception: we must keep "final" code for each author.

**Comment by hurrinico** [comment]
@pedrobaeza take a look if it's ok what i've made

**Review by eLBati — APPROVED** [review]
Once https://github.com/OCA/website/pull/310 is merged

File: website_cookie_notice/views/website.xml
Comment (yajo) [inline website_cookie_notice/views/website.xml]: Please include https://github.com/OCA/website/pull/333

**Review by yajo — CHANGES_REQUESTED** [review]

File: website_cookie_notice/views/website.xml
Comment (eLBati) [reply website_cookie_notice/views/website.xml]: @Yajo added 22067cc5aa3d5ce43f6b0a8e6d73e99d18ede26d

**Comment by JonathanNEMRY** [comment]
https://github.com/hurrinico/website/pull/2 before merge: add setuptools and use manifest instead of openerp

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
I added a commit to make it compatible with #353 when merged.

**Comment by yajo** [comment]
Rebased now that #353 is merged.

**Review by pedrobaeza — APPROVED** [review]
Merging when Travis finishes

**Pull request merged**
